### PR TITLE
styling

### PR DESCRIPTION
**Pull request by ZahraArshia**
Hello 😊 
## in this step of project:
styles are added into project to match [Zeplin project](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) design

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -17,22 +17,33 @@ export default function Book({ book }) {
...
+          </div>
         </div>
         <div>
-          <p>{book.status}</p>
-          <p>{book.currentChapter}</p>
-          <p>{book.chapterNumber}</p>
-          <button type="button">Update progress</button>
+          <p className="status">currentChapter</p>
+          <p>chapterNumber</p>
+          <button className="bookButton" type="button">Update progress</button>
```
Comment (andersonlebon) [inline src/components/Book.js]: - [ ] Kindly note that there is a border that separate the progress bar section with the chapter section.

#### the template 
image

#### Your design
image

File: src/components/AddBook.js [hunk 2]
```diff
@@ -44,6 +44,6 @@ export default function AddBook() {
         </select>
         <button type="submit">Add Book</button>
```
Comment (andersonlebon) [inline src/components/AddBook.js:45]: - [ ] Kindly make sure also this add button has a white color.

image

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @ZahraArshia 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ZahraArshia** [comment]
Hello again 😄 
-  the border between progress bar section and the chapter section is added
- add button text colour changed into white

**Pull request merged**